nemo_rl/models/policy/workers/megatron_policy_worker.py

```
@@ -908,6 +911,8 @@ def train(
         mbs: Optional[int] = None,
     ) -> dict[str, Any]:
         """Train the policy on a batch of data with a given loss function."""
+        worker_start_time = time.time()
```

Ruff (0.14.8)

Local variable `worker_start_time` is assigned to but never used

Remove assignment to unused variable `worker_start_time`

(F841)